frontend/src/organizations/Organizations.js

```diff
@@ -120,7 +120,7 @@ export default function Organizations() {
         />
         <InfoBox
           title={t`HTTPS Configured`}
-          info={t`Shows the percentage of domains which have HTTPS configured and upgrade HTTP connections to HTTPS (ITPIN 6.1.1)`}
+          info={t`Shows the percentage of domains which have HTTPS configured and upgrade HTTP connections to HTTPS`}
```

Comment:
HTTPS is configured and HTTP connections redirect to HTTPS. <a href=link>Websites and Services Hardening requirement 1.1</a>.